utils/configure-rabbitmq

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+set -u
+
+#
+# XXX: this is for communicating between docker containers, the container below listens on localhost:15672
+#
+# docker network create --subnet=172.19.0.0/16 proca || echo "Ok, virtual network 'proca' already exists"
+#
+
+VOL=/tmp/proca-queue
+URL=http://localhost:15672/api
+
+mkdir $VOL || true
+
+echo "--- starting rabbitmq as docker container ------------"
+docker run --detach -p 5672:5672 -p 15672:15672  \
+    -v $VOL:/var/lib/rabbitmq --name rabbitmq    \
+    rabbitmq:3-management
+
+echo "--- waiting 30 seconds for rabbitmq to start ----------"
+
+sleep 30
```

Comment:
Use:
`while ! nc -z localhost 5672; do sleep 0.1; done`

Reply:
I used 0.5 since it takes quite a while to start.